script/utils/constants.json

```diff
@@ -1,181 +1,245 @@
 {
   "chainIds": {
-    "MAINNET": 1,
-    "SEPOLIA": 11155111,
+    "ALEPH_ZERO": 41455,
     "ARBITRUM": 42161,
-    "ARBITRUM_SEPOLIA": 421614,
-    "BSC": 56,
-    "POLYGON": 137,
-    "POLYGON_AMOY": 80002,
-    "ZK_SYNC": 324,
-    "OPTIMISM": 10,
-    "OPTIMISM_SEPOLIA": 11155420,
     "BASE": 8453,
-    "BASE_SEPOLIA": 84532,
-    "LENS": 232,
-    "LENS_TESTNET": 37111,
-    "LINEA": 59144,
-    "LINEA_SEPOLIA": 59141,
-    "SCROLL": 534352,
-    "SCROLL_SEPOLIA": 534351,
-    "UNICHAIN": 130,
-    "UNICHAIN_SEPOLIA": 1301,
-    "ALEPH_ZERO": 41455,
     "BLAST": 81457,
-    "BLAST_SEPOLIA": 168587773,
+    "BOB": 60808,
+    "BSC": 56,
     "BOBA": 288,
     "INK": 57073,
-    "INK_SEPOLIA": 763373,
+    "LENS": 232,
+    "LINEA": 59144,
     "LISK": 1135,
-    "LISK_SEPOLIA": 4202,
+    "MAINNET": 1,
     "MODE": 34443,
-    "MODE_SEPOLIA": 919,
+    "OPTIMISM": 10,
+    "POLYGON": 137,
     "REDSTONE": 690,
+    "SCROLL": 534352,
     "SONEIUM": 1868,
+    "SUPERSEED": 5330,
+    "UNICHAIN": 130,
     "WORLD_CHAIN": 480,
-    "ZORA": 7777777
+    "ZK_SYNC": 324,
+    "ZORA": 7777777,
+    "SOLANA": 34268394551451,
+    "ARBITRUM_SEPOLIA": 421614,
+    "BASE_SEPOLIA": 84532,
+    "BLAST_SEPOLIA": 168587773,
+    "BOB_SEPOLIA": 808813,
+    "INK_SEPOLIA": 763373,
+    "TATARA": 129399,
+    "LENS_SEPOLIA": 37111,
+    "LISK_SEPOLIA": 4202,
+    "MODE_SEPOLIA": 919,
+    "OPTIMISM_SEPOLIA": 11155420,
+    "POLYGON_AMOY": 80002,
+    "SCROLL_SEPOLIA": 534351,
+    "SEPOLIA": 11155111,
+    "UNICHAIN_SEPOLIA": 1301,
+    "ZK_SYNC_SEPOLIA": 300,
+    "SOLANA_DEVNET": 133268194659241
   },
   "oftEids": {
-    "ARBITRUM": 30110,
-    "ARBITRUM_SEPOLIA": 40231,
-    "BASE": 30184,
-    "BASE_SEPOLIA": 40245,
-    "BLAST": 30243,
-    "BSC": 30102,
     "MAINNET": 30101,
     "OPTIMISM": 30111,
-    "OPTIMISM_SEPOLIA": 40232,
+    "BSC": 30102,
+    "UNICHAIN": 30320,
     "POLYGON": 30109,
+    "LENS": -1,
+    "BOBA": -1,
```

Comment:
ooc, rather than enumerating each chain w/ -1 for no OFT EID, could we just omit chains without an OFT EID?